packages/web3-constants/evm/token.json

```diff
@@ -17,7 +17,9 @@
         "Celo": "0x471EcE3750Da237f93B8E339c536989b8978a438",
         "Fantom": "0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83",
         "Aurora": "0xC9BdeEd33CD01541e1eeD10f90519d2C06Fe3feB",
-        "Aurora_Testnet": ""
+        "Aurora_Testnet": "",
+        "Harmony": "",
```

Comment:
<img width="488" alt="image" src="https://user-images.githubusercontent.com/1141198/158512586-aa094f7b-8011-4e18-946c-3022002c5504.png">

WONE should be the wrapped native token.